app/models/item.rb

```diff
@@ -0,0 +1,6 @@
+class Item < ApplicationRecord
+  belongs_to :merchant
+
+  validates_presence_of :name, :description, :price, :image, :inventory, :merchant_id
+  validates_inclusion_of :active_status, :in => [true,false]
```

Comment:
ahh nice i see you're using the inclusion of here, this is good! Forget about my comment above about this.